pandas/core/window/numba_.py

```diff
@@ -0,0 +1,100 @@
+import types
+from typing import Any, Callable, Dict, Optional, Tuple
+
+import numpy as np
+
+from pandas._typing import Scalar
+from pandas.compat._optional import import_optional_dependency
+
+
+def make_rolling_apply(func, args, nogil, parallel, nopython):
```

Comment:
Not a blocker here since this is large enough, but would be nice to annotate this in a follow up